tests/test_all.cc

```diff
@@ -19,6 +19,7 @@
 #include "test_glszm.h"
 #include "test_ngtdm.h"
 #include "test_roi_blacklist.h"
+#if 0

```

Comment:
any particular reason for disabling these tests?

Reply:
Good find! I overlooked reenabling that fragment.
During development, I have to disable stuff referencing Apache on my development machine every time because it fails my builds and ALLEXTRAS/NOEXTRAS does not fully isolate me from Apache dependencies. 
By the way, where we are at plans to separate Nyxus I/O into a library binary in order not to compile the infrastructure and platform specific code every time Nyxus' algorithmic part is compiled?